chia/plot_sync/receiver.py

```diff
@@ -299,7 +312,13 @@ def to_dict(self, counts_only: bool = False) -> Dict[str, Any]:
             "failed_to_open_filenames": get_list_or_len(self._invalid, counts_only),
             "no_key_filenames": get_list_or_len(self._keys_missing, counts_only),
             "duplicates": get_list_or_len(self._duplicates, counts_only),
+            "total_plot_size": self._total_plot_size,
+            "syncing": {
+                "initial": self.initial_sync(),
+                "plot_files_processed": self.current_sync().plots_processed,
+                "plot_files_total": self.current_sync().plots_total,
```

Comment:
This is `self`, you don't need the accessors?

Reply:
See 9baa49f68a3343451345a6fb5fbe9a4d2777137a